src/content/blog/astro-5130.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: 'Astro 5.13'
+description: 'Astro 5.13 ships stuff!'
+homepageLink:
+  title: 'Astro 5.13'
+  subtitle: 'Available now!'
+publishDate: '2025-08-14'
+authors:
+  - sarah
+  - ema
+coverImage: '/src/content/blog/_images/astro-5130/blog-post-cover-astro-5130.webp'
+socialImage: '/src/content/blog/_images/astro-5130/og-astro-5130.webp'
+lang: 'en'
+related:
+  - astro-5120
+  - astro-5110
+---
+
+import Mention from '/src/components/Mention.astro';
+
+**🍀 Lucky Astro 5.13 brings stuff, stuff, and more!**
+
+The starting lineup includes:
+
+- [**Experimental static `import.meta.env`**](#expermental-static-importmetaenv)
+- [**Experimental Chrome DevTools workspace support**](#experimental-chrome-devtools-workspace-support)
+- [**Multiple sitemaps**](#multiple-sitemaps)
+- [**Experimental hosted error pages with the Node.js adapter**](#experimental-hosted-error-pages)
+- [**Enum support in Astro DB tables**](#enum-support-in-astro-db-tables)
+- [**Local support for Cloudflare KV**](#local-support-for-cloudflare-kv)
+
+To upgrade an existing project, use the automated `@astrojs/upgrade` CLI tool. Alternatively, upgrade manually by running the upgrade command for your package manager:
+
+```sh
+# Recommended:
+npx @astrojs/upgrade
+
+# Manual:
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## Experimental static `import.meta.env`
+
+Adds an experimental flag `staticImportMetaEnv` to disable replacement of `import.meta.env` values with `process.env` calls and their coercion, and supersedes the `rawEnvValues` experimental flag
+
+Astro allows you to configure a [type-safe schema for your environment variables](https://docs.astro.build/en/guides/environment-variables/#type-safe-environment-variables), and converts variables imported via `astro:env` into the expected type.
+
+However, Astro by default turns non-public `import.meta.env` values into `process.env` calls during the build, if the environment variable name is present in `process.env`. It also converts your environment variables used through `import.meta.env` in some cases, and this can prevent access to some values, such as the strings `"true"` (which is converted to a boolean value), and `"1"` (which is converted to a number).
+
+The `experimental.staticImportMetaEnv` flag disables this behavior, ensuring that `import.meta.env` values are always inlined.
+
+To enable this feature, add the experimental flag in your Astro config:
+
+```diff
+import { defineConfig } from "astro/config";
+
+export default defineConfig({
++  experimental: {
++    staticImportMetaEnv: true
++  }
+});
+```
+
+This experimental flag supersedes the `experimental.rawEnvValues` flag, which you need to remove in your Astro config:
+
+```diff
+import { defineConfig } from "astro/config";
+
+export default defineConfig({
+   experimental: {
+-    rawEnvValues: true
++    staticImportMetaEnv: true
+   }
+});
+```
```

Comment:
``````suggestion
To enable this feature, add the experimental flag in your Astro config and remove the `experimental.rawEnvValues` flag if it was enabled:

```js title="astro.config.mjs" ins={4,5,7} del={6}
import { defineConfig } from "astro/config";

export default defineConfig({
   experimental: {
     rawEnvValues: true,
     staticImportMetaEnv: true,
   }
});
```
``````